kolibri/plugins/document_epub_render/assets/src/views/EpubRendererIndex.vue

```diff
@@ -734,10 +714,14 @@
   $navigation-button-normal: 52px;
 
   .epub-renderer {
+    @extend %dropshadow-1dp;
```

Comment:
rather than a 1dp drop shadow, let's make this a 1px solid line:

![image](https://user-images.githubusercontent.com/2367265/60469524-d4cb9680-9c11-11e9-8ee8-fd404d4874a4.png)

Reply:
which color token?

Reply:
used the grey-300 color